wca-integration/src/main/java/eu/itesla_project/wca/WCAImpl.java

```diff
@@ -339,6 +343,8 @@ private static void copyRequired(Path workingDir) throws IOException {
                             network.getStateManager().setWorkingState(curativeStateId);
                             AmplUtil.fillMapper(mapper, network); // because action can create a new bus
                             new AmplNetworkWriter(network, dataSource, contingencyNum, curativeActionNum, true, mapper, CLUSTERS_AMPL_EXPORT_CONFIG).write();
+                            if (config.isExportStates())
+                                new WCAStateExporter(network, workingDir, contingencyNum, curativeActionNum).export();
```

Comment:
Add braces to respect java coding guideline even if there is only one statement.